src/test/java/com/vritant/oms/web/rest/PriceListResourceIntTest.java

```diff
@@ -175,6 +225,27 @@ public void getPriceList() throws Exception {
 
     @Test
     @Transactional
+    public void getPriceListPrices() throws Exception {
+        // Initialize the database
+        priceListRepository.save(priceList);
+        millRepository.save(mill);
+        qualityRepository.save(q);
+        sgsRepository.save(sgs);
+        priceRepository.saveAndFlush(price);
+
+        // Get the priceList
+        restPriceListMockMvc.perform(get("/api/priceLists/{id}", priceList.getId()))
+            .andExpect(status().isOk())
+            .andExpect(content().contentType(MediaType.APPLICATION_JSON))
+            .andExpect(jsonPath("$.id").value(priceList.getId().intValue()))
+            .andExpect(jsonPath("$.pricess[0].value").value(2.3d))
+            .andExpect(jsonPath("$.pricess[0].mill.id").value(mill.getId().intValue()))
+            .andExpect(jsonPath("$.pricess[0].quality.id").value(q.getId().intValue()))
+            .andExpect(jsonPath("$.pricess[0].simpleGsmShade.id").value(sgs.getId().intValue()));
+    }
+
+    @Test
+    @Transactional
```

Comment:
@Transactional ? basically runs the test in a transaction.
for more details http://docs.oracle.com/javaee/7/api/javax/transaction/Transactional.html